src/test/java/com/neo4j/docker/TestConfSettings.java

```diff
@@ -264,27 +264,28 @@ void testEnterpriseOnlyDefaultsConfigsAreSet () throws Exception
         }
     }
 
-    @Test
-    void testCommunityDoesNotHaveEnterpriseConfigs() throws Exception
-    {
-        Assumptions.assumeTrue(TestSettings.EDITION == TestSettings.Edition.COMMUNITY,
-                               "This is testing only COMMUNITY EDITION configs");
-        Path debugLog;
-        try(GenericContainer container = createContainer().withEnv("NEO4J_dbms_memory_pagecache_size", "512m"))
-        {
-            //Mount /logs
-            Path logMount = HostFileSystemOperations.createTempFolderAndMountAsVolume( container, "logs-enterprisesettingsnotincommunity-", "/logs" );
-            debugLog = logMount.resolve( "debug.log" );
-            SetContainerUser.nonRootUser( container );
-            //Start the container
-            container.setWaitStrategy( Wait.forHttp( "/" ).forPort( 7474 ).forStatusCode( 200 ) );
-            container.start();
-        }
+//    @Test
```

Comment:
Is this test no longer valid?

Reply:
the test checks that in community edition we aren't adding any enterprise specific configurations (like for clustering). It used to be that they were both the same, and community edition would always have warnings about invalid settings. 

Anyway, now that there are no enterprise specific settings there's nothing to actually test for. So, yes the test is not valid any more, but it could become valid again in the future?